mysql/datadog_checks/mysql/mysql.py

```diff
@@ -985,6 +985,9 @@ def _get_stats_from_innodb_status(self, db):
                 "Privilege error or engine unavailable accessing the INNODB status tables (must grant PROCESS): %s", e,
             )
             return {}
+        except (UnicodeDecodeError, UnicodeEncodeError) as e:
+            self.warning("Unicode error while getting INNODB status: %s", e)
```

Comment:
This will show a warning in the Datadog UI as well - might be noisy. Is this intended, or is a warning in the Agent logs enough?

Also, should we add a note in the warning about whether this is a problem? (I understand not so much unless it happens very frequently)

```suggestion
            self.warning("Unicode error while getting INNODB status (if this warning is infrequent, metric collection won't be impacted): %s", e)
```

Reply:
The warning in the Agent logs might be enough indeed.

I'll also add a note.